db/seeds.rb

```diff
@@ -27,3 +30,34 @@
 item16 = coffee.items.create!(title: 'Cocoa', description: 'delicious organic chocolate to be savoured as it is or combined with intriguing infusions.', price: 500, image: File.new("#{Rails.root}/app/assets/images/coco.jpg"))
 item17 = coffee.items.create!(title: 'Latte', description: 'A shot of strong espresso coffee, with a healthy covering of hot steamed milk and a topping of steamed milk froth served in a glass.', price: 500, image: File.new("#{Rails.root}/app/assets/images/latte.jpg"))
 item18 = coffee.items.create!(title: 'Green Tea', description: 'Green tea potential health benefits for everything from fighting cancer to helping your heart, and it taste pretty okay too.', price: 500, image: File.new("#{Rails.root}/app/assets/images/green-tea.jpg"))
+
+
+jeff_order_1 = item1.orders.create(total_cost: item1.price, user_id: jeff.id, status: 'completed')
+jeff_order_1.item_orders.last.update(quantity: 10, line_item_cost: item1.price)
+
+jeff_order_2 = item3.orders.create(total_cost: item3.price, user_id: jeff.id, status: 'ordered')
+jeff_order_2.item_orders.last.update(quantity: 11, line_item_cost: item3.price)
+
+jeff_order_3 = item7.orders.create(total_cost: item7.price, user_id: jeff.id, status: 'cancelled')
+jeff_order_3.item_orders.last.update(quantity: 15, line_item_cost: item7.price)
+
+jeff_order_4 = item10.orders.create(total_cost: item10.price, user_id: jeff.id, status: 'paid')
+jeff_order_4.item_orders.last.update(quantity: 19, line_item_cost: item10.price)
+
+rachel_order_1 = item2.orders.create(total_cost: item2.price, user_id: rachel.id, status: 'completed')
```

Comment:
Line is too long. [102/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.